server/app.js

```diff
@@ -71,7 +71,7 @@ const sessionOptions = {
   saveUninitialized: true,
 };
 
-if (prod) {
+if (prod && process.env.REDIS_URL) {
```

Comment:
redis is needed to store sessions. If it's not properly configured, we should crash the app (optionally on NODE_ENV=develop, we could just display a big red warning)